include/core/CCompressedLfuCache.h

```diff
@@ -174,6 +174,18 @@ class CCompressedLfuCache {
         return false;
     }
 
+    void clear() {
+        this->guardWrite(DONT_TIME_OUT, [&] {
+            m_NumberLookups = 0;
+            m_NumberHits = 0;
+            m_LostCount = 0;
+            m_RemovedCount = 0;
```

Comment:
I decided to reset all the counters to zero here.  The alternative would be to increment the removed count instead to indicate everything being removed.  It's fairly academic anyway, as we don't expose these counters to the Java side - they are only for our unit tests at the moment.